April Fools/config.yml

```diff
@@ -0,0 +1,21 @@
+UseAltVoice: true
+ResponseConfig:
+  StartedTracking:
+    - Text: |-
+        <voice gender='female'>Hey {{User}}, I have something very important to tell you. Google Has dedicated 1 Billion Dollars to giving the world <break strength='weak'/> Wait, it seems like there's something even more important.
+        This just in...  Aliens have landed on Mount Wayupthar. They have hidden items in Hyrule and Zebes and have been ordered by Ganon and Mother Brain, the leaders, to take over the two worlds.
+        They have cpatured Shaktool, of Maridia, whoe has enough diabetes supplies to last 48 hours. Someone must save him and the world from these evil invaders.</voice>
```

Comment:
cpatured

Reply:
whoe

Reply:
These are actually typos from the original game, so they were left in intentionally. That being said, 99% of folks wouldn't recognized that, so probably not worth keeping in.

Reply:
In that case, maybe it _is_ worth keeping in for the people that _would_ recognize it. It's still just basic TTS anyway, so most other people might just assume it's the TTS fucking up.

Reply:
Actually, j/k. The game has typos (check you feet for example), but this was apparently a typo from the place that I grabbed the transcript from. lol I'll have to be diligent in cross referencing betus's LP.